### PR TITLE
add domain 'youjizz'

### PR DESCRIPTION
**Pull request by MuenYu**

### CODE REVIEW

File: data/youjizz [hunk 1]
```diff
@@ -0,0 +1,2 @@
+youjizz.com
+beta.youjizz.com
```
Comment (DuckSoft) [inline data/youjizz:2]: `youjizz.com` covers `beta.youjizz.com`

**Comment by EpLiar** [comment]
Only second level domains need to be collected and only cdn will use full name.

So you should remove beta.youjizz.com.

Another thing, remember to sort the file content.

**Comment by DuckSoft** [comment]
You may also want to include:
```
cdne-static.yjcontentdelivery.com
cdne-pics.yjcontentdelivery.com
abre-videos.yjcontentdelivery.com
...
```

**Comment by EpLiar** [comment]
> You may also want to include:
> 
> ```
> 
> cdne-static.yjcontentdelivery.com
> 
> cdne-pics.yjcontentdelivery.com
> 
> abre-videos.yjcontentdelivery.com
> 
> ...
> 
> ```

Only `yjcontentdelivery.com`.

**Pull request merged**